CHANGELOG.md

```diff
@@ -2,6 +2,18 @@
 
 Records breaking changes from major version bumps
 
+## 37.0.0
+
+PR [#379](https://github.com/alphagov/digitalmarketplace-utils/pull/379)
+
+The FE apps should now import Flask dependencies via dmutils. For example:
+
+```python
+from dmutils import flask, flask_featureflags, flask_login, flask_wtf
```

Comment:
Have just been playing around with this and it feels like it's going to be a pretty big change. Anywhere we do something like `from flask import Markup` will break and we'll have to do `from dmutils import flask` and then reference it as `flask.Markup` everywhere. Either that, or we keep importing directly from `flask` and ignore this (which is what I suspect will happen...), which for us doesn't technically matter as they're the same thing - but semantically it feels like we're making a change that makes things more difficult and will introduce confusion. I'm torn on this one now... 🤔

Reply:
Maybe we make a new repository called `digitalmarketplace-core` for all of these 'centralised' things :trollface:

Reply:
digitalmarketplace-core
_and_
digitalmarketplace-frontend-core :trollface: 💥

Reply:
We should be able to do `from dmutils.flask import Markup` though

Reply:
Hmm, I tried it and that didn't work. 🤷‍♂️ 

I vaguely recall having issues with importing deep dependencies, but I can't really recall the mechanics behind it.

Reply:
How about I split the `__init__` stuff into a separate PR and just have the dependency update in this one?

Reply:
Yeah I think that's a good idea before this one gets bogged-down.

Reply:
I don't *really* understand why we need to import this *through* `dmutils`, just because the dependency is fulfilled by dmutils requirements. There are a good number of places we directly import packages that are second-tier dependencies, I think this sets a weird precedent.

Reply:
Where and why do we do that?
Say you have 
Package A that depends on Module B and Module C
Module B depends on Module C

So we remove Module C from Package A's requirements...

What then happens if Module B drops Module C as a dependency?

Reply:
Indeed hypothetically it's a wrongness, but it's done a lot because the reality is that most packages don't expose *their* dependencies in a reliable way. An example - we only depend on `Flask-WTF` yet we freely import from `wtforms` - and I'm not totally sure `flask_wtf` actually exposes all of what's accessible through `wtforms`.

Reply:
We should import Flask through `dmutils` because that best reflects how we're collecting the package. _Through_ the requirements of `dmutils`.

![img_20180413_095657](https://user-images.githubusercontent.com/3469840/38726140-400ae5d2-3f01-11e8-95e5-e93ffd033aee.jpg)

Reply:
Right, but this isn't a pattern that can be followed in the general case, because as I mentioned above, the vast majority of packages in the python world don't follow this strictly enough, so it would be a "rule" we're breaking almost everywhere else.